src/components/Home.js

```diff
@@ -29,43 +32,53 @@ const Home = ({ handleTokenCreation, setToken, setCollectionId }) => {
   };
 
   return (
-    <main>
-      <button onClick={handleTokenCreation} className="add-button">
-        Create a Shopping List
-      </button>
-
-      <div>
-        <h1>Join an existing shopping list by entering a token</h1>
-      </div>
+    <div className="wrapper">
+      <div className="app">
+        <div className="app__dots-shadow panel">
+          <main className="align-center">
+            <Header />
+            <div>
+              <button
+                onClick={handleTokenCreation}
+                className="app__create-button"
+              >
+                Create a New List
+                <span className="create-button__icon">
+                  <ArrowIcon />
+                </span>
+              </button>
 
-      <form className="form">
-        <div className="form-row">
-          <div className="form-group">
-            <label htmlFor="existing-token">Share Token</label>
-            <input
-              type="text"
-              id="existing-token"
-              className="form-field"
-              placeholder="Enter three word token"
-              value={value}
-              onChange={handleChange}
-            />
-          </div>
+              <form className="app__token-form flow">
+                <label htmlFor="token" className="visually-hidden">
+                  Join existing list with a token
+                </label>
+                <div className="token-form__field">
+                  <input
+                    type="text"
+                    id="token"
+                    className="token-form__input"
+                    placeholder="Join with a token"
+                    value={value}
+                    onChange={handleChange}
+                  />
+                  <button
```

Comment:
Love this button and functionality!!! Something I noticed was that it could be interesting to remove the button text 'join with a token' on click (so as soon as the button is active) rather than when the first letter of a token is typed in. And then if the button is clicked out of and there's no text, it could default back to the original message.

Something to explore if you're interested!

Reply:
@alexlsalt, good idea - now that I've updated it, it looks much better. Thank you for the suggestion.